java-squid/src/main/java/org/sonar/java/resolve/JavaSymbol.java

```diff
@@ -324,11 +324,14 @@ public Scope typeParameters() {
     }
 
     public String getFullyQualifiedName() {
-      String ownerName = "";
-      if (!owner.name.isEmpty()) {
-        ownerName = owner.name + ".";
+      // nested class
+      Type type = owner.type();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "type" which hides the field declared at line 65. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)